cli/tests/e2e/snapshots/test_metrics/test_metrics_payload/pro_flag0/metrics-payload.json

```diff
@@ -56,7 +56,7 @@
         "size": 300
       }
     ],
-    "maxMemoryBytes": 48693248,
+    "maxMemoryBytes": 42336256,
```

Comment:
This one is annoying. I've seen it changed a few times; can't we just mask it?

Reply:
https://github.com/semgrep/semgrep/pull/9759